### DISCUSSION TITLE
完全剛接觸電腦科學的可以看一下這個

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#464] [General]
這個系列介紹了電腦的發展歷史、從底層往上大致講解電腦的架構及其如何運作。後續還延伸了各種主題，包括資料結構及演算法、網路、機器學習等。影片有中文及英文字幕，可依個人需求選擇使用。若是剛接觸電腦的人，相信看完了會對這堂課所使用的各種名詞更加熟悉，也能更了解電腦科學這門領域。
https://www.bilibili.com/video/BV1EW411u7th/?spm_id_from=333.337.search-card.all.click
https://www.youtube.com/watch?v=tpIctyqH29Q&list=PL8dPuuaLjXtNlUrzyH5r6jN9ulIgZBpdo&ab_channel=CrashCourse

**Comment by boson13579** [comment]
推推.
就算學過計概的也可以挑一些部份來看看，像裡面有機器人(Robot),計算機視覺(Computer vision),自然語言處理(Nature Language Process)等等，不會太難，也十分有趣
強烈建議看中文的，別對自己的英文太有自信!

**Comment by manipulateed** [comment]
謝謝分享，提供了一個電腦的學習路程和基礎的打底。

**Comment by HsuChen88** [comment]
謝謝分享 配飯吃剛剛好

**Comment by williamyen0623** [comment]
推！學習科技領域的知識，計算機科學的概念對整體科技發展和運作的了解非常有幫助，感謝分享！

**Comment by lx3211** [comment]
推推 有些名詞就算老師講解過了還是很陌生，但是看過相個影片就會更加瞭解一些，也不會那麼令人卻步。

**Comment by Chenszuhong** [comment]
感謝分享，這對於初學者的我感覺還不錯

**Comment by limitx0** [comment]
感謝分享，這些影片可以讓我對電腦科學有更多的了解

**Comment by Ben6uo** [comment]
太神了 這個真的對我這個電腦白痴太有用了

**Comment by RogelioKG** [comment]
Crash Course Computer Science
它的前幾堂課會簡單介紹一些電腦底層的運作原理，
其實大概就是計算機概論前四章會提到的內容，
如果沒有修過計概但是又有在寫程式的，
還蠻推薦去看一下的。

**Comment by Yilin20021016** [comment]
非常實用 這是一個對電腦小白有很大幫助的東西
